client/lib/abtest/index.js

```diff
@@ -311,7 +310,7 @@ ABTest.prototype.assignVariation = function () {
 	let variationName, randomAllocationAmount;
 	let sum = 0;
 
-	const userId = get( user, 'data.ID' );
+	const userId = user()?.get()?.ID;
```

Comment:
No need to optionally chain `user().get()`. Only the `ID` property access should be optional.

Reply:
See above.